SimTracker/TrackerMaterialAnalysis/plugins/TrackingMaterialProducer.cc

```diff
@@ -159,6 +164,21 @@ void TrackingMaterialProducer::update(const BeginOfTrack* event) {
                             << std::endl;
     }
   }
+
+  //For the HFnose case:
+  //restrict the outher radius to eta 3.3 since there is HGCAL shadowing
+  //restrict the innner radius to eta 4 since it's non projective
+
+  if (isHFNose && track->GetTrackStatus() != fStopAndKill && fabs(track->GetMomentum().eta()) > 3.3 &&
```

Comment:
"3.3" and "4" are also recalled later at L237-238
Better to have them named constants.

(The same also holds for the "2.0" and "2.4" of the ```HGCal``` case right above)

Reply:
added static constexpr float